tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
     "jsx": "react-native",
-    "lib": ["dom", "esnext"],
+    "lib": ["dom", "es5", "es6"],
```

Comment:
This was me flailing trying to get that `LayoutRectangle` type to get recognized…